src/Community/clustering.jl

```diff
@@ -44,10 +45,10 @@ julia> clustering_coefficient(g, Local([1, 2, 3]))
  1.0
  0.0
 
- julia> clustering_coefficient(star_graph(4))
+ julia> clustering_coefficient(SimpleGraph(Generators.Star(4)))
```

Comment:
We have `using LightGraphs.Generators` above. Can't we then just write
```julia
clustering_coefficient(SimpleGraph(Star(4)))
```
?

Reply:
Not yet, because it hasn't been exported.